### DISCUSSION TITLE
오늘 오후 3시를 기점으로 차단당했습니다

### DISCUSSION THREAD

**Discussion by Dddddddssd** [changdoc/naver-vaccine-macro#155] [General]
엣지, 웨일을 서로 명의 다르게 해서 12시부터 각각 지도 창이랑 원하는 병원 창만 띄웠는데 전혀 이상이 없다가 
3시에 한 브라우저가 아예 블락되더니 20분뒤에 다른 브라우저도 블락이 되었습니다.

아예 지도도 띄우지말고 병원 창 한 개만 띄워야 하나요? ㅠㅠ
지도는 블락이 되지 않고 잘 갱신되고, 주기는 건들지 않았습니다.

**Comment by changdoc** [comment]
지도에서 잔여 백신이 발생했다면 신청 페이지로 이동을 하게 되는데 그 타이밍이 짧은시간에 여러 병원의 예약 신청 페이지에 접근하게 되었다고 판단되면 차단 당할 가능성이 있긴하겠네요.
